sqlobject/src/test/java/org/jdbi/v3/sqlobject/statement/TestSqlScripts.java

```diff
@@ -36,10 +36,10 @@ public void testCreateTable() {
 
     private interface Scriptacular {
         // tag::scripts[]
-        @SqlScript("CREATE TABLE <name> ( pk int primary key )")
+        @SqlScript("CREATE TABLE <name> (pk int primary key)")
         void createTable(@Define String name);
 
-        @SqlScript("INSERT INTO cool_table VALUES ( 5 ), ( 6 ), ( 7 )")
+        @SqlScript("INSERT INTO cool_table VALUES (5), (6), (7)")
```

Comment:
I'm surprised/amazed checkstyle is finding these spaces _inside_ the strings.

Reply:
Hmm, that's my bad, sorry

Reply:
I'm ok with normalizing SQL formatting so long as it's not something the test cares about.